SetupDataPkg/Tools/GenCfgData.py

```diff
@@ -501,7 +501,7 @@ def parse(self, parent_name='', curr=None, level=0):
                             # for now
                             cfg_hdr = OrderedDict()
                             cfg_hdr['length'] = '0x04'
-                            cfg_hdr['value'] = '{0x01:2b, (_LENGTH_%s_/4):10b, %d:4b, 0:4b, %s:12b}' %\
+                            cfg_hdr['value'] = '{0x01:2b, (_LENGTH_%s_):10b, %d:4b, 0:4b, %s:12b}' %\
```

Comment:
is there a check that length does not exceed the 10 bits allocated to it?

Reply:
there was not. But there is now :)